paper-chips-section.html

```diff
@@ -53,8 +53,32 @@
 				/**
 				* Array of labels, e.g. [ 'Apples', 'Pears', 'Oranges' ]
 				*/
-				labels: Array
+				labels: Array,
+				/**
+				* Establishes if chips inside the section are selectable
+				*/
+				selectable: {
+					type: Boolean,
+					value: false,
+				},
+			},
+
+			_tapLabel(e) {
```

Comment:
I think we try to use `_onEvent`-naming now (I know the code this came from didn't do that, but I fixed that by now ...)

Reply:
Done, thanks!